tools/release.go

```diff
@@ -50,6 +50,8 @@ var versions = map[string]string{
 	"detectors/gcp/": stable,
 
 	"propagator/": unstable,
+
+	"e2e-test-server/": unstable,
```

Comment:
Adding this fixed an error with the new e2e-test-server/cloud_functions package:
```
$ make prepare-release
echo "make sure tools/release.go is updated to your desired stable and unstable versions"
make sure tools/release.go is updated to your desired stable and unstable versions
go run tools/release.go prepare
go: errors parsing e2e-test-server/cloud_functions/go.mod:
e2e-test-server/cloud_functions/go.mod:8:2: require github.com/GoogleCloudPlatform/opentelemetry-operations-go/e2e-test-server: version "v" invalid: must be of the form v1.2.3
2023/01/30 17:04:37 running "/usr/lib/google-golang/bin/go mod edit -require=github.com/GoogleCloudPlatform/opentelemetry-operations-go/detectors/gcp@v1.11.0 e2e-test-server/cloud_functions/go.mod": exit status 1
exit status 1
make: *** [Makefile:215: prepare-release] Error 1

```